build.gradle.kts

```diff
@@ -8,5 +8,15 @@ configurations {
 }
```

Comment:
Should the it come with the Kotlin lint module and the .editorconfig file?

Reply:
A very good question, I wasn't sure if we are definitely adopting ktlint or not.  I will find out and stick it in a future PR, thanks.